packages/arb-ts/token_lists/uniswapDefaultList.ts

```diff
@@ -0,0 +1,343 @@
+export default [
```

Comment:
Instead of committing the uniswap list, we could have a bash script that downloads it then runs the token list gen against the downloaded file

Reply:
my thinking is to hardcode it to limit the amount of surprise; if we were to dynamically fetch the uniswap list (or some other), i'd like that to be a separate step done prior to generate json, which can add (but maybe in a l8er PR)